.ci/tritonbench/triton-ptxas.patch

```diff
@@ -6,14 +6,10 @@ index 58264f29d..6c0209871 100644
              fmad = [] if opt.enable_fp_fusion else ['--fmad=false']
              arch = sm_arch_from_capability(capability)
              opt_level = ['--opt-level', '0'] if knobs.nvidia.disable_ptxas_opt else []
-+            if not knobs.nvidia.disable_ptxas_opt and (ptxas_knobs := os.environ.get("PTXAS_KNOBS", None)):
-+                dna_abs_path = os.path.abspath(ptxas_knobs)
-+                if not os.path.exists(dna_abs_path):
-+                    raise FileNotFoundError(f"PTXAS_KNOBS file not found at {dna_abs_path}")
-+                if dna_abs_path.endswith(".txt") or dna_abs_path.endswith(".json"):
-+                    opt_level = ["--plain-text-controls", f"--apply-controls={dna_abs_path}"]
-+                else:
-+                    opt_level = [f"--apply-controls={dna_abs_path}"]
++            if not knobs.nvidia.disable_ptxas_opt and (ptxas_options := os.environ.get("PTXAS_OPTIONS", None)):
++                kernel_name = os.environ.get("PTXAS_OPTIONS_KERNEL", None)
++                if not kernel_name or kernel_name == metadata["name"]:
++                    opt_level = [ptxas_options.split(" ")]
```

Comment:
Should it be just `ptxas_options.split(" ")` ? Is `opt_level: List[str]` or is it `opt_level: List[List[str]]` ? 
```suggestion
+                    opt_level = ptxas_options.split(" ")
```